src/Story/Story.php

```diff
@@ -25,12 +25,11 @@ final public function __construct (
 
 	/**
 	 */
-	final public function getId () : int
+	final public function getUuid () : string
```

Comment:
My bad.. uuid is way more useful, as it is more widely used inside of storyblok

Reply:
If I remember correctly, the `id` field is also considered deprecated and supersed by their Uuid, which makes sense. So that's a good change anyhow :)